src/DiadocHttpApi.CloudCryptAsync.cs

```diff
@@ -65,6 +65,15 @@ public Task<AsyncMethodResult> DssSignAsync(string authToken, string boxId, DssS
 			return PerformHttpRequestAsync<AsyncMethodResult>(authToken, "POST", queryString.BuildPathAndQuery(), Serialize(request));
 		}
 
+		[Obsolete("Currently unavailable. Use DssSignAsync")]
+		public Task<AsyncMethodResult> DssSignV2Async(string authToken, string boxId, DssSignRequestV2 request)
+		{
+			var queryString = new PathAndQueryBuilder("V2/DssSign");
```

Comment:
Для однообразия с остальными методами

```suggestion
			var queryString = new PathAndQueryBuilder("/V2/DssSign");
```